importer/balanced/builder.go

```diff
@@ -8,8 +23,11 @@ import (
 	ipld "gx/ipfs/Qme5bWv7wtjUNGsK2BNGVUFPKiuxWrsqrtvYwCLRw8YFES/go-ipld-format"
 )
 
-func BalancedLayout(db *h.DagBuilderHelper) (ipld.Node, error) {
-	var offset uint64 = 0
+// Layout builds a balanced DAG. Data is stored at the leaves
+// and depth only increases when the tree is full, that is, when
+// the root node has reached the maximum number of links.
+func Layout(db *h.DagBuilderHelper) (ipld.Node, error) {
```

Comment:
Be aware there are two or three API changes due to `stuttering`.

Reply:
Does golint complain about `stuttering` or is that just a personal preference.

Reply:
It complains about it